openedx/core/djangoapps/discussions/__init__.py

```diff
@@ -0,0 +1 @@
+default_app_config = 'openedx.core.djangoapps.discussions.apps.DiscussionsConfig'
```

Comment:
Would it be possible to uniform the naming here? There are multiple `discussion` and `discussions` references, which is confusing.

Reply:
The location of this code is temporary. There will be decision about that part later.